### ISSUE TITLE
(#250) Build & theme teams view and node

### ISSUE THREAD

**Issue by nadia847** [MantaRayMedia/alnap#120]
###Done criteria
- [x] Build and theme team node (for staff members) #146
- [x] Can add multiple people to a team
- [x] Can add team as a nugget on another page
- [x] Team nugget works as grid
- [x] Team nugget works as slideshow
- [x] Team nugget works as list (1 item per row)

### ALNAP review criteria
[team page](http://alnap-dev.mrmdev.co.uk/about/meet-our-team/london-team)
- [ ] Can add multiple people to a team
- [ ] Can add team as a nugget on another page
- [ ] Team nugget works as grid
- [ ] Team nugget works as slideshow

**Comment by nadia847** [comment]
@ican-icant @dave-manta Is there anything non-remora to do with these?

**Comment by dave-manta** [comment]
@nadia847 could you rephrase your question, as I'm not sure how to answer.

**Comment by nadia847** [comment]
@dave-manta are the requirements for people and teams in any way different from remora functionality?

**Comment by dave-manta** [comment]
I believe that Remora functionality will suffice @nadia847.

**Comment by angievanhegan** [comment]
I'm moving this back to the backlog as it's not within the current sprint goals (and there's already plenty in the sprint for the client). It's ready for ALNAP review.

**Comment by Maria-alnap** [comment]
@nadia847 should I be checking this now?

**Comment by Maria-alnap** [comment]
@nadia847 We've tested this and have replied on issue #146 

(@matty-manta not sure we've been shown how to do add/view team nuggets on other pages yet so I can't check that)

**Comment by matty-manta** [comment]
@Maria-alnap we can take a look at this in our skype chat this afternoon, when you're finished with Dave perhaps?
